### PR TITLE
Use in-repository updated copy of lektor-atom plugin instead of lektor-atom 0.3 which uses a deprecated Werkzeug's atom

### PR DESCRIPTION
**Pull request by dzc0d3r**
Use **in-repository** updated **copy of lektor-atom plugin** instead of lektor-atom v0.3  which uses a deprecated Werkzeug's atom api 

This replaces Werkzeug's Atom with Pelican's feedgenerator
this work is based on [this PR to fix deprecared Werkzeug's atom ](https://github.com/nixjdm/lektor-atom/pull/25)

**Also** this should fix [issue #388](https://github.com/beeware/beeware.github.io/issues/388) **ModuleNotFoundError: No module named 'werkzeug.contrib'**

🎉🎉🎉🚀🚀❤️❤️

### CODE REVIEW

**Comment by dzc0d3r** [comment]
@freakboy3742 please take a look at this

**Comment by freakboy3742** [comment]
@dzc0d3r My apologies - I've been buried in other work over the last few days. I'll take a look right now.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,7 +17,8 @@ jobs:
         pip install --upgrade pip
         pip install --upgrade setuptools
         # lektor-atom 0.3 uses an API deprecated in Werkzeug 1.0
-        pip install "Werkzeug<1.0"
+        # pip install "Werkzeug<1.0"
+        # currently using in-repositry updated copy of lektor-atom plugin
```
Comment (freakboy3742) [inline .github/workflows/ci.yml:21]: There three comment lines are no longer needed.

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -19,7 +19,8 @@ jobs:
         pip install --upgrade pip
         pip install --upgrade setuptools
         # lektor-atom 0.3 uses an API deprecated in Werkzeug 1.0
-        pip install "Werkzeug<1.0"
+        # pip install "Werkzeug<1.0"
+        # currently using in-repositry updated copy of lektor-atom plugin
```
Comment (freakboy3742) [inline .github/workflows/publish.yml:23]: These three comment lines are no longer needed.

File: packages/lektor_atom_plugin/lektor_atom.py [hunk 3]
```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
+
+"""
+This is an updated copy of lektor-atom plugin
+wich replace deprecated Werkzeug's atom with Pelican's feedgenerator .
+"""
```
Comment (freakboy3742) [inline packages/lektor_atom_plugin/lektor_atom.py:6]: Minor typos: "which replaces the deprecated Werkzeug's atom with Pelican's feedgenerator."

File: packages/lektor_atom_plugin/setup.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+"""
+This is an updated copy of lektor-atom plugin
+wich replace deprecated Werkzeug's atom with Pelican's feedgenerator .
```
Comment (freakboy3742) [inline packages/lektor_atom_plugin/setup.py:5]: Minor typos: "which replaces the deprecated Werkzeug's atom with Pelican's feedgenerator."

**Comment by freakboy3742** [comment]
I've just pushed those updates. I've worked out what happened with permissions, too - you've opened this PR against the root of your repository. Since that's "your copy" of the project, maintainers are prevented from pushing to that repository (by default, unless you give them explicit access as you've done). 

For future reference, if you create a *branch* for any PRs, the maintainer will (by default) be able to push to the branch.

**Pull request merged**